numpy/linalg/linalg.py

```diff
@@ -1810,7 +1810,7 @@ def det(a):
 
 # Linear Least Squares
 
-def lstsq(a, b, rcond=-1):
+def lstsq(a, b, rcond="warn"):
```

Comment:
Thought about that at some point, but then decided that the NoValue thingy actually has a very specific purpose for functions dispatching to methods, and "warn" gives a much clearer idea when looking at the signature.